okta.json

```
@@ -3084,6 +3093,7 @@
             "description": "Lists all roles assigned to a user",
             "type": "investigate",
             "identifier": "list_roles",
+            "verbose": "Below role is need to run this action.<ui><li>SUPER_ADMIN</li></ui>",
```

if we're going to still use a list here despite one item, we may as well be consistent and still say:

```suggestion
            "verbose": "One of the following roles is required to run this action:<ui><li>SUPER_ADMIN</li></ui>",
```

same for subsequent cases of the same text

---

Changed as per suggestion.